assets/styles/style.css

```diff
@@ -738,7 +738,7 @@ body {
 	font-size: 16px;
 }
 
-#bbpress-forums h3 {
+#bbpress-forums>h3 {
```

Comment:
I think it's easier to read if there are spaces around the `>`.

Reply:
Done.